regtest/docker-compose.yml

```diff
@@ -0,0 +1,653 @@
+    <<: *base-lnd
+    container_name: boltz-lnd-2
+    platform: linux/amd64
+    hostname: lnd-2
+    ports:
+      - 11009:10009
+    volumes:
+      - lnd2-data:/root/.lnd/
+    profiles: ["default", "ci"]
+
+  bitcoind-init:
+    <<: *base-scripts
+    platform: linux/amd64
+    restart: no
+    container_name: boltz-bitcoind-init
+    entrypoint:
+      ["/bin/bash", "-c", "source /etc/profile.d/utils.sh && bitcoind-init"]
+    depends_on:
+      - bitcoind
+    profiles: ["default", "ci"]
+
+  regtest-start:
+    <<: *base-scripts
+    platform: linux/amd64
+    restart: no
+    container_name: boltz-regtest-start
+    entrypoint:
+      ["/bin/bash", "-c", "source /etc/profile.d/utils.sh && regtest-start"]
+    depends_on:
+      bitcoind-init:
+        condition: service_completed_successfully
+      elementsd:
+        condition: service_healthy
+      lnd-1:
+        condition: service_healthy
+      lnd-2:
+        condition: service_healthy
+      cln-1:
+        condition: service_healthy
+      cln-2:
+        condition: service_healthy
+      anvil:
+        condition: service_healthy
+      rif-relay:
+        condition: service_healthy
+    profiles: ["default", "ci"]
+
+  scripts:
+    <<: *base-scripts
+    restart: no
+    platform: linux/amd64
+    container_name: boltz-scripts
+    depends_on:
+      boltz-backend:
+        condition: service_healthy
+    profiles: ["default", "ci"]
+
+  electrs:
+    hostname: electrs-bitcoin
+    container_name: boltz-electrs-bitcoin
+    restart: always
+    platform: linux/amd64
+    image: ${ELECTRS_IMAGE}
+    depends_on:
+      bitcoind-init:
+        condition: service_completed_successfully
+    entrypoint: ["electrs-bitcoin"]
+    command:
+      - --electrum-rpc-addr
+      - electrs:19001
+      - --http-addr
+      - electrs:3002
+      - --daemon-rpc-addr
+      - bitcoind:18443
+      - --network
+      - regtest
+      - --jsonrpc-import
+    healthcheck:
+      test: ["CMD-SHELL", "curl -s $(hostname):3002/blocks/tip/height"]
+      timeout: 1s
+      retries: 20
+      interval: 2s
+      start_period: 5s
+    ports:
+      - 19001:19001
+      - 3002:3002
+    volumes:
+      - bitcoin-data:/root/.bitcoin
+    profiles: ["default", "ci"]
+
+  electrs-liquid:
+    hostname: electrs-liquid
+    platform: linux/amd64
+    container_name: boltz-electrs-liquid
+    restart: always
+    image: ${ELECTRS_IMAGE}
+    depends_on:
+      elementsd:
+        condition: service_healthy
+    entrypoint: ["electrs-liquid"]
+    command:
+      - --electrum-rpc-addr
+      - electrs-liquid:19002
+      - --http-addr
+      - electrs-liquid:3003
+      - --daemon-rpc-addr
+      - elementsd:18884
+      - --daemon-dir
+      - /root/.elements
+      - --cookie
+      - regtest:regtest
+      - --network
+      - liquidregtest
+      - --parent-network
+      - regtest
+      - --jsonrpc-import
+    healthcheck:
+      test: ["CMD-SHELL", "curl -s $(hostname):3003/blocks/tip/height"]
```

Comment:
I got this error: 
```
invalid interpolation format for services.electrs-liquid.healthcheck.test.[]: "curl -s $(hostname):3003/blocks/tip/height". You may need to escape any $ with another $
```
```suggestion
      test: ["CMD-SHELL", "curl -s $$(hostname):3003/blocks/tip/height"]
```

Reply:
Applied in [06e22a9](https://github.com/breez/breez-sdk-liquid/pull/707/commits/06e22a9abfc961e056783a35ddd1fae696502426)